test_contact_manual.py

```
@@ -0,0 +1,218 @@
+                return False
+        
+        print("  ✓ Rate limiter allows 5 requests")
+        
+        # 6th request should be rate limited
+        allowed = check_rate_limit(test_ip)
+        if allowed:
+            print("  ✗ Rate limiter should have blocked 6th request")
+            return False
+        
+        print("  ✓ Rate limiter blocks 6th request")
+        return True
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)